src/templates/artworks.js

```diff
@@ -47,6 +49,9 @@ class ArtworksPage extends React.Component {
 render() {
   const data = this.props.data;
   const { frontmatter } = data.markdownRemark;
+  const url = location.pathname;
```

Comment:
Maybe i could create an utility function to retrieve the langKey values, i have this piece of code in different part of the project.